src/main/fox_hud.c

```diff
@@ -2284,6 +2360,10 @@ void func_8008B2A4(void) {
     }
 }
 
+s32 D_800D1EB4 = 255;
+s32 D_800D1EB8 = 255;
+s32 D_800D1EBC = 255;
```

Comment:
I almost suggested a chain assign here before I realized that wouldn't work at all.